### DISCUSSION TITLE
Any way to "reset" or bulk delete images in Frigate+?

### DISCUSSION THREAD

**Discussion by grayslawson** [blakeblackshear/frigate#13874] [Ask A Question]
I have a paid Frigate+ account and I recently completely reconfigured all my cameras, including buying new models and completely changing where they are set up at.  I'd like to clear out all my old images used for training my model but I don't see any option to bulk/mass delete them, and clicking each one, clicking delete, then verifying I want to delete each one is going to take countless hours.

Is there any way to start fresh?  Or to bulk delete images inside the Frigate+ website?  Thank you!

**Comment by NickM-27** [comment]
Email blake@frigate.video
